extensions/agent/src/dialogs/pickScheduleDialog.ts

```diff
@@ -38,7 +39,17 @@ export class PickScheduleDialog {
 	}
 
 	public async showDialog() {
-		await this.model.initialize();
+		this.model.initialize().then((result) => {
```

Comment:
Again - please don't use .then. Just add the code inside it below the await this.model.initialize() call and it'll do the same thing.

Reply:
This is intended. `.then()` works differently compared to `await`. We don't want to wait and block the UI for a backend job. This is what the PR is fixing.

Reply:
Ah, yeah ok. This method doesn't have to be async then it looks like